src/pages/settings/Payments/PaymentsPage/index.native.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
```

Comment:
I'm struggling to spot the differences between this file and the web implementation, which is probably a sign we can DRY it up.

Reply:
+1 I think maybe we just need to get the positional information for the touch so we can set the anchor position? Maybe there's something else... but I can't tell.

Reply:
Yea, the only difference I see is in `paymentMethodPressed`. If that's the case, we could create a module exporting the platform specific method and consolidate `index.js` and `index.native.js`.

Reply:
Yeah the only difference is the positional information for the anchor. A module is probably a good idea